### DISCUSSION TITLE
about eclipse debugger

### DISCUSSION THREAD

**Discussion by yangmeng66** [eclipse-platform/eclipse.platform.releng.aggregator#902] [Q&A] [answered]
hello, This link：  https://www.eclipse.org/articles/Article-Debugger/how-to.html

Is this document ready? Where can I find it
![image](https://user-images.githubusercontent.com/51857567/221567211-435d8ba4-b7b2-497c-815b-90f0fe4373dd.png)

**Comment by iloveeclipse** [comment]
Quote from the article (emphasis mine):

> And in our third paper (**not yet published**), Enhancing the Eclipse Debugger

I assume it is either not in the **open** Internet (the article is coming from IBM debug team) or available but with a different title. 
There is no one left in the project from original authors of the cited article who could confirm that, unfortunately.

**Reply by akurtakov** [reply]
Probably this section should be deleted. I don't think that this is in a repo maintained by the project so I'll summon @fredg02 for information where this articles relies in git.

**Reply by laeubi** [reply]
I also have already missed that "enhanced article" so if there is any chance to get this from somwhere...

**Reply by fredg02** [reply]
:genie:  => https://git.eclipse.org/c/www.eclipse.org/articles.git/tree/Article-Debugger/how-to.html

I don't know if the "Enhancing the Eclipse Debugger" article was ever published.

**Reply by akurtakov** [reply]
As none of us knows anything about it @fredg02 how can " And in our third paper (not yet published), Enhancing the Eclipse Debugger, we describe how to enhance the UI for our assembly language debugger by adding all the little bells and whistles that one expects in modern IDEs." be removed from this article aka where does this page lives?

**Reply by yangmeng66** [reply]
First of all, thank you for your reply. If this article has not been published, are there any other reference documents about eclipse debugger?

**Reply by laeubi** [reply]
> are there any other reference documents about eclipse debugger?

You can **debug the code** with the _eclipse debugger_ to see how the debugger works :-)

**Reply by fredg02** [reply]
> where does this page lives?

@akurtakov The link above points to a Git/Gerrit repository. Or did you mean something else?

**Reply by akurtakov** [reply]
Sorry! I missed the link.

**Comment by akurtakov** [comment, marked as answer]
https://git.eclipse.org/r/c/www.eclipse.org/articles/+/200210 removes the note from the article to not cause confusion anymore. Whoever has the rights please push.
